--- node_modules/@apollo/server/dist/esm/ApolloServer.js.orig	2024-07-22 21:51:10
+++ node_modules/@apollo/server/dist/esm/ApolloServer.js	2024-07-22 21:51:36
@@ -1,7 +1,7 @@
 import { isNodeLike } from '@apollo/utils.isnodelike';
 import { InMemoryLRUCache, PrefixingKeyValueCache, } from '@apollo/utils.keyvaluecache';
 import { makeExecutableSchema } from '@graphql-tools/schema';
-import resolvable from '@josephg/resolvable';
+import resolvable from '@josephg/resolvable/index.js';
 import { GraphQLError, assertValidSchema, print, printSchema, } from 'graphql';
 import loglevel from 'loglevel';
 import Negotiator from 'negotiator';
\ No newline at end of file
